hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Constants.java

```diff
@@ -1327,7 +1327,7 @@ private Constants() {
    * The special S3 region which can be used to talk to any bucket.
    * Value {@value}.
    */
-  public static final String AWS_S3_CENTRAL_REGION = "us-east-1";
```

Comment:
there is already a default region below.

Reply:
kept both constants, with `AWS_S3_CENTRAL_REGION = AWS_S3_DEFAULT_REGION`